Add documentation on importing a package from a branch (#6)
kurtosis-tech/package-template-repo: README.md

> this_package = import_module("github.com/YOURUSER/THISREPO/main.star")

Reviewer: ```suggestion
# For remote packages:
this_package = import_module("github.com/YOURUSER/THISREPO/main.star") 

# For local packages:
this_package = import_module(".src/main.star")
```

```

> If you want to import the package from a branch other than main, you can update the import command to:

Reviewer: ```suggestion
If you want to use a fork or specific version of this package in your own package, you can replace the dependencies in your `kurtosis.yml` file using the [Replace](https://docs.kurtosis.com/concepts-reference/kurtosis-yml/#replace) primitive. 
```

> 

Reviewer: ```suggestion
Within your `kurtosis.yml` file:
```

```python

> this_package = import_module("github.com/YOURUSER/THISREPO/main.star@YOURBRANCH")

Reviewer: ```suggestion
name: github.com/example-org/example-repo
replace:
    github.com/YOURUSER/THISREPO: github.com/YOURUSER/THISREPO@YOURBRANCH
```
